src/main/java/es/unizar/smartcampuz/infrastructure/service/RootController.java

```diff
@@ -0,0 +1,22 @@
+package es.unizar.smartcampuz.infrastructure.service;
```

Comment:
I think this class is not needed. It seems like a test class or something like that. I don't see any injection of this class in your new code, as well as any request to that endpoint.